[5.7] PXC-3951: Current computation of disk used with sst-idle-timeout could misidentify SST as idle

venkatesh-prasad-v opened the pull request:
https://jira.percona.com/browse/PXC-3951

Problem
---
Currently, disk usage computation of sst-idle-timeout is:

        du --max-depth=1 ${tmpsstdir} | tail -n 1 | awk '{print $1}'

This logic does not detect the storage size difference in bytes, thereby
causing the SST script to timeout and eventually terminate SST even when the
SST is in progress.

Solution
---
The script now passes `--bytes (-b)` and `--summarize (-s)` to `du` to
get difference of storage in bytes.

altmannmarcelo commented on scripts/wsrep_sst_xtrabackup-v2.sh:
```diff
@@ -306,7 +306,7 @@ monitor_sst_progress() {
     fi
 
-    current_size=$(du --max-depth=1 ${tmpsstdir} | tail -n 1 | awk '{print $1}')
+    current_size=$(du -b -s ${tmpsstdir} | tail -n 1 | awk '{print $1}')
```
do we need `| tail -1` if we use --summarize and only ${tmpsstdir} as target ?

venkatesh-prasad-v replied:
Removed `| tail -n 1`

altmannmarcelo reviewed (approved):
apart from tail, PR looks good. You can decide what to do (keep or remove) and it's fine to me either way.

Commit:
PXC-3951: Current computation of disk used with sst-idle-timeout could misidentify SST as idle

https://jira.percona.com/browse/PXC-3951

Problem
=======
Currently, disk usage computation of sst-idle-timeout is:

        du --max-depth=1 ${tmpsstdir} | tail -n 1 | awk '{print $1}'

This logic does not detect the storage size difference in bytes, thereby
causing the SST script to timeout and eventually terminate SST even when the
SST is in progress.

Solution
========
The script now passes `--bytes (-b)` and `--summarize (-s)` to `du` to
get difference of storage in bytes.
